src/library_pthread.js

```diff
@@ -962,7 +962,7 @@ var LibraryPThread = {
 
   $proxyToMainThread__deps: ['$withStackSave', '_emscripten_run_on_main_thread_js'].concat(i53ConversionDeps),
   $proxyToMainThread__docs: '/** @type{function(number, (number|boolean), ...number)} */',
-  $proxyToMainThread: (index, sync, ...callArgs) => {
+  $proxyToMainThread: (funcIndex, codePtr, sync, ...callArgs) => {
```

Comment:
Perhaps add a comment here about the new role of 0 for the first and second parameters? Skimming the code it seems like the idea might be that one of the two must be 0 and the other nonzero?

Reply:
I've added more comments.   There are somewhat redundant between `proxyToMainThread` and `emscripten_receive_on_main_thread_js` since they are two sides of the same mechanism.